content/en/case-studies/_index.html

```diff
@@ -63,6 +61,9 @@ <h4><i>"Kubernetes has the opportunity to be the new cloud platform. The amount
     <main>
         <h3>Kubernetes Users</h3>
         <div id="usersGrid">
+         
+          <a target="_blank" href="/case-studies/ing/"><img src="/images/case_studies/ing_feature.png" alt="ING"></a>
+          <a target="_blank" href="/case-studies/capital-one/"><img src="/images/case_studies/capitalone_feature.png" alt="Capital One"></a>
           <a target="_blank" href="/case-studies/northwestern-mutual/"><img src="/images/case_studies/northwestern_feature.png" alt="Northwestern Mutual"></a>
```

Comment:
Could you double check that all the of logos in the bottom section are the same size. I think the latter 3 in the first row are larger.